### ISSUE TITLE
build skipped when test code changes during "pulp test"

### ISSUE THREAD

**Issue by clinuxrulz** [purescript-contrib/pulp#123]
When I change the non-test code "pulp build" and "pulp test" both trigger a build.

However when I add a test case (only changing the test code), then run "pulp test", the build is not triggered for the test code.

I change test code then go.

```
> pulp test
* Project unchanged; skipping build step.
* Running tests...
V Passed: test1
* Tests OK.
```

**Edit:** This is for pulp version 5.0.2

**Comment by hdgarrood** [comment]
In the purescript branch, this was addressed simply by having `pulp test` never attempt to skip the build. I think this is probably the right approach, because `psc` already builds incrementally; the build skipping is to allow us to avoid the slow things like `psc-bundle` or `browserify`, and `pulp test` doesn't do either of these things. So I think this is fixed in the purescript branch.

**Comment by hdgarrood** [comment]
Fixed as of 6.0.0.
